reflex/components/__init__.py

```diff
@@ -100,7 +100,6 @@
 editable_input = EditableInput.create
 editable_preview = EditablePreview.create
 editable_textarea = EditableTextarea.create
-editor = Editor.create
```

Comment:
still want to keep the new editor exposed.

Reply:
Good catch my bad